testsuite/features/step_definitions/salt_steps.rb

```diff
@@ -416,6 +416,10 @@
 When(/^I enter the MAC address of "([^"]*)" in (.*) field$/) do |host, field|
   if host == 'pxeboot-minion'
     mac = $pxeboot_mac
+  elsif host == 'ubuntu-minion'
```

Comment:
Can we use `host.include? 'ubuntu'`  instead?
I'm thinking of using that step in different test envs, like QAM, where we can have other ubuntu hosts.

Reply:
Sure

Reply:
Done